docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,39 +1,34 @@
+* Added enabling Map WAN Reference (https://github.com/hazelcast/hazelcast-platform-operator/pull/216[#216])
+* Deleted dependent CRs when a Hazelcast CR is marked to be deleted (https://github.com/hazelcast/hazelcast-platform-operator/pull/314[#314])
+* Added support for stopping HotBackup gracefully (https://github.com/hazelcast/hazelcast-platform-operator/pull/318[#318])
+* Added skipping the check for a Management Center lock file  (https://github.com/hazelcast/hazelcast-platform-operator/pull/333[#333])
+* Added Phone Home as a separate process (https://github.com/hazelcast/hazelcast-platform-operator/pull/336[#336])
+* Added Phone Home data collection for Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/341[#341])
+* Added Phone Home data collection for WAN Replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/342[#342])
+* Added Phone Home data collection for UCD (https://github.com/hazelcast/hazelcast-platform-operator/pull/344[#344])
+* Added Phone Home data collection for Backup & Restore (https://github.com/hazelcast/hazelcast-platform-operator/pull/345[#345])
 
 == Fixed Issues
 
-* Fixed the problem where the LoadBalancer hostname is present but not resolvable by
-the clients. Wait for DNS to resolve AWS ELB hostnames (https://github.com/hazelcast/hazelcast-platform-operator/pull/169[#169])
-* Fixed wrong duration unit usage in client connectionStrategy, use correct
-seconds (https://github.com/hazelcast/hazelcast-platform-operator/pull/174[#174])
-* Set HotBackup status to the failure when HotBackup fails (https://github.com/hazelcast/hazelcast-platform-operator/pull/198[#198])
-* Fixed evictionPolicy field name (https://github.com/hazelcast/hazelcast-platform-operator/pull/210[#210])
-* Fixed the issue that caused scheduled backups to continue even after the `scheduled` field was removed from the `HotBackup` custom resource. (https://github.com/hazelcast/hazelcast-platform-operator/pull/218[#218])
-
+* Omit the throwing "Reconciler error" at the beginning of the Hazelcast platform setup (https://github.com/hazelcast/hazelcast-platform-operator/pull/323[#323])
+* Fixed hot backup issue when the status was successful before the external backup uploading had finished (https://github.com/hazelcast/hazelcast-platform-operator/pull/242[#242])
+* Fixed the issue when HotBackup with BucketURI will trigger backup upload for the Local backup type (https://github.com/hazelcast/hazelcast-platform-operator/pull/261[#261])
+* Fixed timeout issue for long-running backup upload in platform-operator-agent (https://github.com/hazelcast/hazelcast-platform-operator/pull/288[#288])
+* Fixed error on enabling persistence without PVC (https://github.com/hazelcast/hazelcast-platform-operator/pull/339[#339])
+* Fixed operator idle issue when it didn't pick up created CRs (https://github.com/hazelcast/hazelcast-platform-operator/pull/343[#343])
```

Comment:
```suggestion
* Fixed an operator issue which caused it to remain idle instead of picking up new CRs  (https://github.com/hazelcast/hazelcast-platform-operator/pull/343[#343])
```